### ISSUE TITLE
Show only playing music channel

### ISSUE THREAD

**Issue by mintyleaf** [Kentakoong/MusicBar#13]
When you listening from one source, then starting playing another - all good
But when pausing it, while first is playing - paused one is stuck

So we have to check somehow if current source is active or not

**Comment by Kentakoong** [comment]
yeah, this would be really hard to do, since it only fetches the latest one. 
Would be a great idea if we could do it in the future.
